content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
       text: 'Why should I install the Datadog Agent on my cloud instances?'
 ---
 
+This guide provides an introduction to the Agent and how you can use it to pull system level metrics into the Datadog Platform. It also provides a walkthrough of an example Agent installation on an Ubuntu environment which will cover:
+
+  - Agent Installation
+  - Verifying that the Agent is running
+  - Configuring Agent features
+  - Troubleshooting Resources
+
 ## Overview
 
-The Agent is lightweight software installed on your hosts. It reports metrics and events from your host to Datadog using [integrations][1], [DogStatsD][2], or the [API][3]. With additional setup, the Agent can report [live processes][4], [logs][5], and [traces][6].
+### About the Agent
+
+The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. It can run on your local hosts (Windows, MacOS), containerized environments (Docker, Kubernetes), on premises data centers, and supports configuration management tools (Chef, Puppet, Ansible). The Agent is able to collect 75-100 system level metrics every 15-20 seconds. With additional setup and configuration, the Agent can also report live processes, logs, and traces to the Datadog Platform. The Datadog Agent is open source and its source code is available on GitHub at [DataDog/datadog-agent][1].
```

Comment:
```suggestion
The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. It can run on your local hosts (Windows, MacOS), containerized environments (Docker, Kubernetes), and in on-premises data centers. You can install and configure it using configuration management tools (Chef, Puppet, Ansible). 

The Agent is able to collect 75 to 100 system level metrics every 15 to 20 seconds. With additional setup and configuration, the Agent can also send live processes, logs, and traces to the Datadog Platform. The Datadog Agent is open source and its source code is available on GitHub at [DataDog/datadog-agent][1].
```